### ISSUE TITLE
Eslint *.vue file support

### ISSUE THREAD

**Issue by programmador** [wsdjeg/SpaceVim#4488]
## Problem
See screenshot below - seems like eslint plugin propagates some hidden (from me) options which make eslint to treat *.vue files as *.html files.

* Eslint Spacevim plugin works perfectly in spacevim for other projects where TypeScript is split into separate file for Vue components.
* Eslint CLI works perfectly for the current project, so I can lint (and also fix!) in terminal
* I've also tried the same project in Spacemacs - everything is working fine
* My colleague uses Webstorm with eslint plugin - it also works perfectly without any additional project config or npm package installation

The last 2 items are mentioned to make sure that the problem is not in project config.

`eslint-cli` is installed:
```
> npm -g list
/usr/lib
├── eslint-cli@1.1.1
├── ...
```

## Expected behavior, english is required
Spacevim should report linter errors on file save.
It should not require any additional plugins

## The reproduce ways from Vim starting (Required!)

* install layers: vue, javascript, typescript
* open *.vue file in any project
* make a change
* save the file
* open `:messages`

## Output of the `:SPDebugInfo!`
Sorry pasting it here breaks issue markup. Anyway I don't think it's useful for current case.

## Screenshots

![screenshot_2021-10-01-171402](https://user-images.githubusercontent.com/674995/135635812-143f4b6d-d33b-42b5-9c01-86249a18e0df.png)

**Comment by wsdjeg** [comment]
I can not reproduce this issue now. more steps are needed. which project do you test it?

**Comment by wsdjeg** [comment]
Ok, because the default linter of vue use `['--plugin', 'html']` as argvs. maybe this should be changed. or at least should be documented.
--------------
wsdjeg

**Comment by wsdjeg** [comment]
fixed, you need to run `npm install -g eslint eslint-plugin-vue`

**Comment by wsdjeg** [comment]
If you still got can not find plugin `eslint-plugin-vue`, please checkout:

https://github.com/yannickcr/eslint-plugin-react/issues/2339
--------------
wsdjeg

**Comment by programmador** [comment]
`eslint eslint-plugin-vue` is already installed into the project (non-globally).

I've reproduced the same bug in CLI by adding --plugin=html as You've mentioned.
So yes, the problem was in the wrong argument.

BTW I've tried with `--plugin=vue` and it works like a charm (in CLI).
I've also tried `--plugin=abcde` and it fails so it seems like `vue` is really a valid option though frankly I didn't dig too much.

Anyway thank You very much, now it works perfectly!
